src/views/orders_view.js

```diff
@@ -0,0 +1,11 @@
+import Backbone from 'backbone';
+
+const OrdersView = Backbone.View.extend({
```

Comment:
We should probably remove this file because we already have `src/views/order_list_view.js` which defines an `OrdersView` (and that is the file that we're using in `src/app.js`).